awscli/clidriver.py

```diff
@@ -656,10 +657,15 @@ def invoke(self, service_name, operation_name, parameters, parsed_globals):
             value is returned.
 
         """
+        if hasattr(parsed_globals, 'read_timeout'):
```

Comment:
`test_paginate_rc` and `test_ctrl_c_is_handled` in `test_clidriver`. It doesn't look like the namespace is being patched, so I don't know why it isn't showing up.